### ISSUE TITLE
panel.first and panel.last not working as optional plot argument in abacus_plot

### ISSUE THREAD

**Issue by jdpye** [ocean-tracking-network/glatos#79]
In GitLab by @chrisholbrook on Nov 13, 2019, 15:53

One should be able to add points, lines, etc. to abacus plots using plot arguments panel.first (e.g., plot behind the abacus points). For example: 

```
#get path to example detection file
det_file <- system.file("extdata", "walleye_detections.csv",
  package = "glatos")
det <- read_glatos_detections(det_file)

#subset one transmitter
det2 <- det[det$animal_id == 153, ]


abacus_plot(det2, locations=NULL,
  main = "TagID: 32054", col = "red", panel.first = abline(h=3, col = "green"))

```

This should plot a green horizontal but it does not. To make this work, need to wrap the panel.first expression in `quote()`; e.g., 
```
abacus_plot(det2, locations=NULL,
  main = "TagID: 32054", col = "red", panel.first = quote(abline(h=3, col = "green")))
```
![image](https://my-gitlab-bucket.s3.amazonaws.com/433448100/352005ba36e5a258d555e06db2656e95.png)

Perhaps we need to `substitute()` or `quote()` the `...` arguments before coercing to list. Also, maybe some useful guidance [here](https://stackoverflow.com/questions/13353847/how-to-expand-an-ellipsis-argument-without-evaluating-it-in-r)?

**Comment by jdpye** [comment]
In GitLab by @chrisholbrook on Nov 13, 2019, 16:04

changed the description

**Comment by jdpye** [comment]
In GitLab by @chrisholbrook on Nov 13, 2019, 17:21

fixed in 0.4.1; d9e3d9e

**Comment by jdpye** [comment]
In GitLab by @chrisholbrook on Feb 19, 2020, 21:50

closed
